src/python/grpcio_tests/tests_aio/unit/server_interceptor_test.py

```diff
@@ -123,14 +123,14 @@ async def wrapper(
 
 async def _create_server_stub_pair(
     *interceptors: aio.ServerInterceptor
-) -> Tuple[aio.Server, test_pb2_grpc.TestServiceStub]:
+) -> Tuple[aio.Server, aio.Channel, test_pb2_grpc.TestServiceStub]:
```

Comment:
Can you please explain why this is necessary? I don't see the returned channel being used anywhere. Is this about keeping the reference around?

Reply:
Yes. The Python gRPC Stub only keeps the product of `unary_unary` method (Multicallable objects), which depends on the `cygrpc.AioChannel` instead of `grpc.aio.Channel`. Upon this comment, I found that we can inject a reference of Channel into Multicallable object. This behavior is more desired, because we won't break users that wasn't careful enough about their channels and stubs. PR updated.